<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>behavython</class>
 <widget class="QMainWindow" name="behavython">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Behavython</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background-color:#4B4B4B;color:#FFFFFF;}</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="styleSheet">
    <string notr="true">QWidget{	background-color:#353535;}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scroll_area">
      <property name="styleSheet">
       <string notr="true">QScrollArea{border:0px}QScrollBar:vertical{background:#353535;width:5px}QScrollBar::handle:vertical{background:#2C53A1;min-width:20px;}QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{background:#353535;}QScrollBar:horizontal{background:#353535;height:5px}QScrollBar::handle:horizontal{background:#2C53A1;min-width:20px;}QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal{background:#353535;}</string>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scroll">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>244</width>
         <height>234</height>
        </rect>
       </property>
       <widget class="QFrame" name="interface_frame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{}QCheckBox{color:#FFFFFF;font:8pt&quot;DejaVu Sans&quot;;font-weight:bold;}QCheckBox::indicator{width:15px;height:15px;background-color:#606060;border-radius:4px;}QCheckBox::indicator:checked{background-color:#A21F27;}</string>
        </property>
        <widget class="QPushButton" name="analyze_button">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>180</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{font:10pt&quot;DejaVu Sans&quot;;font-weight:bold;}QPushButton{border:2px solid #A21F27;border-radius:8px;background-color:#2C53A1;color:#FFFFFF;font:10pt&quot;DejaVu Sans&quot;;font-weight:bold;}QPushButton:pressed{border:2px solid #A21F27;border-radius:8px;background-color:#A21F27;color:#FFFFFF;}</string>
         </property>
         <property name="text">
          <string notr="true">ANALYZE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>231</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#2C53A1;border-radius:8px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">SETTINGS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_0">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>231</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#A21F27;border-radius:8px;color:#FFFFFF;font:10pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">BEHAVYTHON </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QFrame" name="image">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>51</width>
           <height>61</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="image_nnc">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>41</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="pixmap">
           <pixmap>NNC_logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="Line" name="line_0">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>231</width>
           <height>10</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QComboBox" name="type_combobox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;background-color:#606060;border:0px;border-radius:6px;}QListView{color:#FFFFFF;font:8pt&quot;Century Gothic&quot;;font-weight:bold;background-color:#969696;border:0px;}QListView{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;background-color:#606060;border:0px;border-radius:6px;}QComboBox::drop-down{width:20px;border:5px;}QComboBox::down-arrow{border-left:2px solid none;border-right:2px solid none;border-top:2px solid #FFFFFF;width:0.5px;height:1px;border-radius:2px;}QComboBox::down-arrow:hover{border-left:2px solid none;border-right:2px solid none;border-top:2px solid #A21F27;width:0.5px;height:1px;border-radius:2px;}QAbstractItemView{border:2px solid #969696;selection-background-color:#2C53A1;}</string>
         </property>
         <property name="currentText">
          <string notr="true">Open Field</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string notr="true">Open Field</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Plus Maze</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="clear_button">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>210</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{font:14pt&quot;DejaVu Sans&quot;;font-weight:bold;}QPushButton{border:2px solid #A21F27;border-radius:8px;background-color:#2C53A1;color:#FFFFFF;font:10pt&quot;DejaVu Sans&quot;;font-weight:bold;}QPushButton:pressed{border:2px solid #A21F27;border-radius:8px;background-color:#A21F27;color:#FFFFFF;}</string>
         </property>
         <property name="text">
          <string notr="true">CLEAR</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="arena_width_lineedit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{border:#969696;background-color:#606060;border-bottom-left-radius:6px;border-top-left-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="arena_height_lineedit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>110</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{border:#969696;background-color:#606060;border-bottom-left-radius:6px;border-top-left-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="frames_per_second_lineedit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{border:#969696;background-color:#606060;border-bottom-left-radius:6px;border-top-left-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">30</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">Experiment type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">Arena width</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">Arena height</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">Frames per second</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>90</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#606060;border-bottom-right-radius:6px;border-top-right-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">cm</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>110</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#606060;border-bottom-right-radius:6px;border-top-right-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">cm</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>130</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#606060;border-bottom-right-radius:6px;border-top-right-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">fps</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="threshold_lineedit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{border:#969696;background-color:#606060;border-bottom-left-radius:6px;border-top-left-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">0.0267</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>150</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border:#969696;background-color:#606060;border-bottom-right-radius:6px;border-top-right-radius:6px;color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">cm/f</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{color:#FFFFFF;font:7pt&quot;DejaVu Sans&quot;;font-weight:bold;}</string>
         </property>
         <property name="text">
          <string notr="true">Threshold</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
